LoopKitUI/Views/Settings Editors/CarbRatioScheduleEditor.swift

```diff
@@ -11,7 +11,7 @@ import HealthKit
 import LoopKit
 
 
-fileprivate extension HKUnit {
+public extension HKUnit {
```

Comment:
I'd prefer that, rather than expose these names, that you create a new extension that defines `gramsPerUnit`